server/services/flood.js

```diff
@@ -188,5 +188,9 @@ module.exports = {
 
   getTargetArea (taCode) {
     return util.getJson(`${serviceUrl}/target-area/${taCode}`)
+  },
+
+  getStationsByRadius (x, y) {
+    return util.getJson(`${serviceUrl}/stations-by-radius/${x}/${y}`)
```

Comment:
If name of function is get by radius, in my mind it should have a configurable parameter for the radius value to be passed through as well. Otherwise name of method should be getStationsWithin8km or something. My preference would be to make the method configurable and pass through the 8000 all the way to the database query.

Reply:
changed to have radius configurable but aslo default to 8k